soc/arm/nxp_imx/rt/soc_rt10xx.c

```diff
@@ -288,26 +288,6 @@ static int imxrt_init(const struct device *arg)
 	/* disable interrupts */
 	oldLevel = irq_lock();
 
-	/* Watchdog disable */
-	if ((WDOG1->WCR & WDOG_WCR_WDE_MASK) != 0) {
-		WDOG1->WCR &= ~WDOG_WCR_WDE_MASK;
-	}
-
-	if ((WDOG2->WCR & WDOG_WCR_WDE_MASK) != 0) {
-		WDOG2->WCR &= ~WDOG_WCR_WDE_MASK;
-	}
-
-	RTWDOG->CNT = 0xD928C520U; /* 0xD928C520U is the update key */
-	RTWDOG->TOVAL = 0xFFFF;
-	RTWDOG->CS = (uint32_t) ((RTWDOG->CS) & ~RTWDOG_CS_EN_MASK)
-		| RTWDOG_CS_UPDATE_MASK;
-
-	/* Disable Systick which might be enabled by bootrom */
-	if ((SysTick->CTRL & SysTick_CTRL_ENABLE_Msk) != 0) {
-		SysTick->CTRL &= ~SysTick_CTRL_ENABLE_Msk;
-	}
-
-	SCB_EnableICache();
```

Comment:
Why was this code removed?

Reply:
This code is a duplicate of what we implement in `SystemInit` for RT10xx SOCs, so it is redundant with this PR.